experimenter/experimenter/experiments/models.py

```diff
@@ -837,6 +837,25 @@ def excluded_live_deliveries(self):
             )
         return matching
 
+    @property
+    def live_experiments_in_namespace(self):
+        matching = []
+        experiment_ids = NimbusBucketRange.objects.filter(
+            isolation_group__name=self.bucket_namespace,
+            isolation_group__application=self.application,
+        ).values_list("experiment_id", flat=True)
+        matching = (
```

Comment:
I would just return this instead of pulling it into a variable.

Reply:
I was just making this change, you read my mind 🫡